krun.py

```diff
@@ -232,7 +229,7 @@ def run(self):
 
         info("Done: Results dumped to %s" % self.out_file)
         if errors:
-            warn("%s ERRORS OCCURRED! READ THE LOG!%s" % (ANSI_RED, ANSI_RESET))
+            warn("Errors occurred! Read the scrollback!")
```

Comment:
If you agree, merge this and I will make a new pull (eventually, I want to spend time on other tasks for a bit now).